src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,54 @@
+.footer_wrapper {
```

Comment:
It is showing 
<img width="1336" alt="image" src="https://github.com/sau-rav/ui-hub-capx/assets/30508582/7b462459-0161-41cc-b25b-a6fcd8bfb492">
 to me while local build.
 Next doesn't support scss file or css file.
 we should use tailwind atomic classes or inline css if requried